CONTRIBUTING.md

```diff
@@ -63,21 +63,24 @@ In particular, look for:
 - [good second issues](https://github.com/python/mypy/labels/good-second-issue)
 - [documentation issues](https://github.com/python/mypy/labels/documentation)
 
-It's also extremely easy to get started contributing to our sister project
-[typeshed](https://github.com/python/typeshed/issues) that provides type stubs
-for libraries. This is a great way to become familiar with type syntax.
-
-If you need help getting started, don't hesitate to ask on
-[gitter](https://gitter.im/python/typing).
+You do not need to ask for permission to work on any of these issues.
+Just [fix the issue yourself, try to add a unit test](#running-tests) and
+[open a pull request](#submitting-changes).
 
 To get help fixing a specific issue, it's often best to comment on the issue
-itself. The more details you provide about what you've tried and where you've
-looked, the easier it will be for you to get help.
+itself. You're much more likely to get help if you provide details about what
+you've tried and where you've looked (maintainers tend to help those who help
+themselves). [gitter](https://gitter.im/python/typing) can also be a good place
```

Comment:
Is it still? I haven't been there in a while

Reply:
I check it fairly regularly. It's quite active, but admittedly it's rare that contributions get discussed there. Usually just user help or potential bug reports. Happy to remove it, but wanted to include an option for attracting attention that isn't just pinging random people.